### DISCUSSION TITLE
Extensions update Task is stuck - Action required

### DISCUSSION THREAD

**Discussion by flow-motion** [akeeba/panopticon#529] [Q&A (Community Support)]
Hi everyone - 
on 4 pages i get the message in the title. I can clear the error, but it comes back. When i update the extensions by hand - mean ONpage - and reload panopticons extension information the bell stays. 
As written: Action required... but what action :) 
What do i have to do?

**Comment by nikosdion** [comment]
Have you updated to 1.1.0 yet? This was a problem with 1.0.7 which I've already fixed.

**Reply by ariesmitskamp** [reply]
probably same issue:
I get this error on (self) updating from 1.0.7 to 1.1.0 from the frontend:
Error:
Undefined constant ZipArchive::RDONLY

  at xxxxxxx/src/Model/Selfupdate.php:352

(php 8.3.2, MariaDB 10.6.16)

**Reply by nikosdion** [reply]
@ariesmitskamp You have a **completely** different issue. Read the OP's description and yours. They are about as similar as bread and bananas 😉 

Ask your host to enable the `zip` extension for the PHP version (8.3.2) you are using with Panopticon. It is a requirement for the self-update feature to work properly.

**Reply by ariesmitskamp** [reply]
@nikosdion I run Panopticon from a vps with some 20 Joomla sites and unzipping has never been a problem. I downloaded 1.1.0, installed from control panel, ran post update actions from updated Panopticon and now have the new version, including Brian's UI stuff. Great!

Keep up the good work!
Thank you,
Arie

**Comment by flow-motion** [comment]
Thanks for your help!
